CDCgov/prime-reportstream
# 9722 - Fix vertical spacing between nav and page content/nav on API Programmer's Guide page

This PR ...

- Fix vertical spacing between nav and page content/nav on API Programmer's Guide page
- Fixed Admin button hover styling.

Test Steps:
1. Log into RS
2. Select Resources -> API Programmers Guide
3. Notice the space between the side nav and the top nav
4. Notice the blue line is in line with the horizontal rule in the header upon hover of the Admin tab

## Changes
![Screenshot 2023-06-29 at 12 32 52 PM](https://github.com/CDCgov/prime-reportstream/assets/102491809/4510f972-65a2-4bf3-9682-f9b32b945792)

![Screenshot 2023-06-29 at 12 32 58 PM](https://github.com/CDCgov/prime-reportstream/assets/102491809/31668c2b-9101-44c5-a5aa-e8eb3e8f7277)

## Checklist

### Testing


## Linked Issues
- Fixes #9722
